src/_component/welcome/welcome.scss

```diff
@@ -0,0 +1,130 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
+
+%circle {
+  height: 40px;
+  width: 40px;
+  border: none;
+  border-radius: 50%;
+}
+
+%font {
+  font-family: 'Roboto';
+  font-weight: 500;
+}
+
+.welcome-page {
+  text-align: center;
+  align-items: center;
+  margin: 0 37px;
+}
+
+.header_title {
+  background: $color-alizarin-crimson;
+  height: 60px;
+  line-height: 60px;
+  @extend %font;
+}
+
+.welcome-page__header {
+  justify-content: center;
```

Comment:
Justify content is not active because display: flex is missing. Consider in future to clean up :)